dnf-behave-tests/features/config.feature

```diff
@@ -172,3 +172,58 @@ Scenario: Lines that contain only whitespaces do not spoil previous config optio
    """
    gpgcheck = 1
    """
+
+
+@bz1721091
+@fixture.httpd
+Scenario: Dnf can use config file from remote location
+  Given I copy repository "miscellaneous" for modification
+    And I create file "/{context.dnf.repos[miscellaneous].path}/remote.conf" with
+    """
+    [repo-from-remote-config]
+    baseurl=http://some.url/
+    """
+    And I use repository "miscellaneous" as http
```

Comment:
Hm, so while this does work, you are abusing the `miscellaneous` repo to set up the HTTP server just to have the config on it. I think we'd be better off adding a simple `I set up a http server for directory <dir>` and use that, because the whole `miscellaneous` repo is redundant in the setup and makes it quite a bit more complex than it needs to be...

Reply:
Well, you are completely right. I tried to sneak this quick and dirty hack in ;). I'll rework the tests the clean way.